.circleci/config.yml

```diff
@@ -335,6 +335,99 @@ jobs:
             git checkout -b release-client-v${VERSION}
             git push -u origin release-client-v${VERSION}
 
+  notify-stuck-stage-nodes-job:
+    resource_class: small
+    docker:
+      - image: cimg/base:2023.01
+    steps:
+      - run:
+          name: Alert Slack of stuck stage nodes
+          command: |
+            handle_error() {
+              # Construct failure Slack message
+              failure_content="{ \"blocks\": ["
+              failure_content+="{ \"type\": \"section\", \"text\": { \"type\": \"plain_text\", \"text\": \"Encountered error while checking for stuck staging nodes\n\" } }"
+              failure_content+="]}"
+              echo "Sending error message to Slack: $failure_content"
+
+              # Send Slack failure message
+              curl -f -X POST -H 'Content-type: application/json' \
+                --data "$failure_content" \
+                $SLACK_DAILY_DEPLOY_WEBHOOK
+            }
+
+            fetchEndpoints() {
+              url=$1
+              fallback=$2
+              fetchedEndpoints=$(curl -s "$url" | jq -r '.data[]' 2>/dev/null)
+
+              if [ -z "$fetchedEndpoints" ]; then
+                echo "FETCH_ERROR"
+                echo "$fallback"
+              else
+                echo $fetchedEndpoints
+              fi
+            }
+
+            (
+              set -e
+
+              # Fetch the latest version from the GitHub repository (assume Content and Discovery have the same latest versions)
+              versionUrl="https://raw.githubusercontent.com/AudiusProject/audius-protocol/main/packages/discovery-provider/.version.json"
+              VERSION=$(curl -s "$versionUrl" | jq -r '.version')
+
+              if [ -z "$VERSION" ]; then
+                echo "Failed to fetch version data"
+                exit 1
+              fi
+
+              contentFallbackEndpoints=("https://creatornode5.staging.audius.co" "https://creatornode6.staging.audius.co" "https://creatornode7.staging.audius.co" "https://creatornode8.staging.audius.co" "https://creatornode9.staging.audius.co" "https://creatornode10.staging.audius.co" "https://creatornode11.staging.audius.co" "https://creatornode12.staging.audius.co")
```

Comment:
This feels worse to me tbh. I'd like to avoid putting anything like this in our CI... why do we need a fall back?

Reply:
like for when the api is down. i don't feel strongly if you want it to just fail and not try the hardcoded list - can make that change now

Reply:
ok fallback is removed completely now. this is what the happy and sad paths look like (i'll unregister that testing node)

<img width="640" alt="Screenshot 2023-12-15 at 3 24 20 PM" src="https://github.com/AudiusProject/audius-protocol/assets/4657956/4529e174-a4c6-42de-bdb2-ac5569ffe614">

Reply:
:D